src/main/java/io/jenkins/plugins/aws/global_configuration/CredentialsAwsGlobalConfiguration.java

```diff
@@ -178,30 +178,24 @@ private Credentials getSessionCredentials(AWSCredentialsProvider credentialsProv
      * @throws IOException
      *             in case ot error.
      */
-    private AWSSessionCredentials sessionCredentialsFromInstanceProfile(@NonNull AwsClientBuilder<?, ?> builder)
+    private AwsSessionCredentials sessionCredentialsFromInstanceProfile(@NonNull AwsClientBuilder<?, ?> builder)
```

Comment:
The automated migration tool failed here and I am just guessing at the correct APIs. Should be manually tested before and after this PR to confirm there are no regressions.

Reply:
Tested manually and confirmed I was able to get credentials from environment variables.